docs/book/src/01_user/01_getting-started.md

```diff
@@ -53,9 +53,11 @@ To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
 No additional steps are required and you can install the RKE2 provider with **clusterctl** directly:
 
 ```bash
-clusterctl init --bootstrap rke2 --control-plane rke2 --infrastructure docker
+clusterctl init --core cluster-api:v1.7.6 --bootstrap rke2:v0.7.0 --control-plane rke2:v0.7.0 --infrastructure docker:v1.7.6
```

Comment:
Clarification: usually when version is not specified ("as-is" before this change), clusterctl tries to fetch the latest released version from the GH repo, however, it was using version `v0.2.4` without this change. I suspect that could be because of org change from `rancher-sandbox` to `rancher` around that release and it was cached. If that is the case, upstream references to new location was updated in https://github.com/kubernetes-sigs/cluster-api/pull/11216